src/CMakeLists.txt

```diff
@@ -198,6 +198,14 @@ install(FILES ${valhalla_hdrs}
   DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/valhalla"
   COMPONENT development)
 
+install(DIRECTORY ${VALHALLA_SOURCE_DIR}/third_party/date/include/date
+  DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/valhalla/third_party"
```

Comment:
this means that any project using valhalla as a library needs to also add this path as an include path. perhaps we should make our headers all say `<valhalla/third_party/whatever_dep>` so that other projects dont need to do that?

Reply:
Hm, the other alternative would be to drop `third_party` and put them in 
```
DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/valhalla/"
```

I'd think it makes more sense to be explicit that something is third_party, like your suggestion.